pkg/state/test-ingress-state_withnamedclasses.yaml

```diff
@@ -0,0 +1,122 @@
+#
+# OCI Native Ingress Controller
```

Comment:
Thinking this is a sequential processing, can we add ingress 'ingress-state-excluded' in the middle section and make sure the last ingress 'ingress-state-new' is processed?